libs.glassfish_logging/external/commons-logging-1.0.4-license.txt

```diff
@@ -0,0 +1,206 @@
+Name: Commons Logging
```

Comment:
This is not Apache Commons Logging, but a shaded version of it. I would stay with the original name, as it was distributed as part of glassfish, same goes then for the description.